### ISSUE TITLE
[Discovery] Foundational knowledge for Mobile VHA clinics

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#25220]
## Problem statement
Mobile health clinics are VA health facilities that are included as locations under the relevant VAMC. These types of facilities were not part of the initial Pittsburgh pilot so the special attributes of this type of location were not foreseen. They don't fit the facility pattern established by the VAMC facility detail pages as they do not offer services at fixed locations. The information needed for these types of facilities may be different from that which is expected for brick and mortar medical clinics. 

Before we reimagine the detail page experience, we want to more fully understand how mobile clinics are being used across the country so that it isn't based on our assumptions (such as: all mobile clinics have a predictable schedule of service locations)

## Background/context

See the [Initiative brief](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/facilities/medical-centers/product/vamc-vha-mobile-redesign/initiative-brief.md)

### Problem to be solved
Before we can reimagine detail pages or validate potential designs with Veterans, we need to better understand these facilities by talking with directors/PAOs. 

### Questions
- What Veteran population is served by mobile clinics?
- Why do Veterans choose mobile clinics for services?
- Are mobile clinics used differently in urban and rural areas?
- Does the mobile clinic have predictable locations/dates/times or is it determined by Veteran demand?
- What services are provided?
- Are appointments necessary? if so, how are they made?
- Is there specific eligibility criteria?
- What information does a Veteran need for these types of facilities?
- What info is meaningful for the "prepare for your visit" section? 

## Tasks
- [x] Prepare email for reaching out to PAO/mobile directors to solicit participation in discovery interviews
- [x] Prepare basic conversation guide for conversations with SMEs (plan to interview director and/or clinician from mobile clinics representing urban and rural areas
- [ ] Interview SME to gain foundational understanding about these facilities

## Acceptance Criteria
- [ ] Convo guide and documentation from SME conversations

**Comment by lildziner** [comment]
- [Mural board](https://app.mural.co/t/vsa8243/m/vsa8243/1621428990239/3d51af4fa995ba023556956d066b2fbdda6a89ff?sender=leydahughes3111) of mobile clinic discovery candidates

- [Email template](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/facilities/medical-centers/product/vamc-vha-mobile-redesign/discovery/email-to-pao.md) to request interviews with PAOs

- [Stakeholder interview questions](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/facilities/medical-centers/product/vamc-vha-mobile-redesign/discovery/stakeholder-interview.md) to serve as a basic conversation guide
